src/util.js

```diff
@@ -3,8 +3,11 @@ export function contains(arrayLike, stringOrRegex) {
 		return arrayLike.indexOf(stringOrRegex) > -1;
 	}
 
-	const match = arrayLike.find(function(item) {
-		return item.match(stringOrRegex);
+	var match;
+	arrayLike.forEach(function(item) {
+		if (item.match(stringOrRegex)) {
+			match = item.match(stringOrRegex);
+		}
```

Comment:
I think previously `find()` would have returned the first match, whereas now `match` will be set to the last match.

Reply:
Agreed! Good catch

Reply:
As strange as it seems, this only returns from the inner `function` -- the `forEach` will still iterate over every entry. There's no easy way to short-circuit a `forEach` (without throwing an exception). I'd recommend a normal `for`-loop:

```
let match;
for (let i=0; i<arrayLike.length; i++) {
    match = arrayLike[i].match(stringOrRegex)) {
    if (match) break;
}
```

Reply:
Could also use `reduce` if you don't care about the early out

```js
const match = arrayLike.reduce(function(memo, item) {
  return memo || item.match(stringOrRegex);
}, undefined);
```